Artiom/src/main/java/Numbers.java

```diff
@@ -0,0 +1,7 @@
+//Display numbers from 1 to 10 using for()
+public class Numbers {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 10; i++)
+            System.out.println(i);
+    }
+}
```

Comment:
Please remove this class from this PR. It is already in your previous PR and will apear here after merge